index.js

```diff
@@ -50,26 +72,60 @@ getUsersCreditDetails(usersArray);
 
 const genderView = users => {
   // Your code goes here ...
+  
+  let femaleUser = users
+    .filter(function(user) {
+      return user.gender == 'female';
+    })
+    .map(function(user) {
+      return user.firstName + ' ' + user.lastName;
+    })
```

Comment:
since you already have a function that does this ( getFullNames() ) you could reuse that instead of the .map